annoFuse/0.1.8/annoFusePerSample.R

```diff
@@ -89,7 +89,7 @@ if(!is_empty(STARFusioninputfile$FusionName) & is_empty(Arribainputfile$"gene1--
 
 
 # General fusion QC for read support and red flags
-fusionQCFiltered<-fusion_filtering_QC(standardFusioncalls=standardFusioncalls,readingFrameFilter="in-frame|frameshift|other",artifactFilter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap|ConjoinG",junctionReadCountFilter=1,spanningFragCountFilter=10,readthroughFilter=FALSE)
+fusionQCFiltered<-fusion_filtering_QC(standardFusioncalls=standardFusioncalls,readingFrameFilter="in-frame|frameshift|other",artifactFilter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap|ConjoinG",junctionReadCountFilter=1,spanningFragCountFilter=100,readthroughFilter=FALSE)
```

Comment:
hmmm, why the change from 10 to 100? I've been out of the loop a while, seems some new stuff has popped up. Also, if a default filter param has changed, then perhaps it should be considered a minor version update? like 0.1.9?

Reply:
With the cutoff of 10 we were able to get ~96% sensitivity for PPTC (100 read length ) data.  However, when we tested TCGA data which has (50-75 read length) the cutoff of 10 wasn't sufficient (~70% sensitivity) but at 100 we get about ~96% sensitivity (percentage overlap to what is previously published for these samples)
<img width="633" alt="Screen Shot 2020-09-11 at 2 30 06 PM" src="https://user-images.githubusercontent.com/34580719/92960363-7ddb6700-f43b-11ea-988d-b830cbf52a39.png">

So we believe we should change the cutoff to 100 to capture all read length datasets , tagging @jharenza.

I can update to the latest tag of the package as well which currently is v0.2, if that seems necessary here.

Reply:
Oooh, interesting! What about the specificity? Are you saying performance stayed good on the original dataset while improving TCGA results? Or should the cutoff move with input read length?

Reply:
So yeah the overlap of high confidence fusion in PPTC dataset remained the same at spanningFragCountFilter == 100 but changing the cutoff to 100 did significantly affect the overlap with TCGA fusion high confidence fusion list.

Technically yes it seems the cutoff is affected by read length but we don't have clear cutoff for each type of read length dataset ( 50,100,150 etc) nor do we see any published paper on that.  So going by TCGA data overlap as reference the 100 cutoff might be best.

About specificity we don't have any true negatives (I think that would only come from like lab experiments to add true negative to the samples not sure ) to calculate that.